src/OpenTelemetry.Api/README.md

```diff
@@ -79,6 +79,11 @@ propagated out of proc using
 [Propagators](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md).
 OpenTelemetry SDK ships a BaggagePropagator and enables it by default.
 
+It is important to note that `Baggage` is not automatically attached to any
+telemetry. User *can* explicitly read `Baggage` and use it to enrich metrics,
+logs and traces. An example of doing this for traces is shown
+[here](../../docs/trace/extending-the-sdk/README.md#processor).
```

Comment:
This link does not show how to enrich traces with `Baggage`.

Reply:
I think it is okay, because # processor has the sub-heading ##enriching processor in the linked doc.